code/modules/detectivework/scanner.dm

```diff
@@ -212,3 +299,30 @@
 	to_chat(user, "<span class='notice'><B>Scanner Report</B></span>")
 	for(var/iterLog in log)
 		to_chat(user, iterLog)
+
+/obj/item/detective_scanner/admin // Strictly an admin tool. returns hidden fingerprints and the duration is instant.
+	name = "badmin forensic scanner"
+	desc = "An unbelievable scanner capable of returning results seemingly out of nowhere."
+	admin = TRUE
+	advanced = TRUE
+	color = "#00FFFF" // aqua
+	scan_icon = FALSE // this scanner doesn't get a chance to play the animation because scans are instant so its best to leave this false to avoid unnecessary icon updates.
+	view_check = FALSE // admin scanner doesn't care if you can actually see something
+
+/obj/item/detective_scanner/admin/Initialize()
+	. = ..()
+	if(prob(50)) // has a 50% chance to either have the regular or advanced sprite. Both with aqua coloring of course
+		icon_state = "forensic2"
+
+/obj/item/detective_scanner/admin/scan(atom/A, mob/user) // plays result sound after scan since it bypasses feedback
+	. = ..()
+	found_something ? feedback(sound_scanner_match , TRUE) : feedback(sound_scanner_nomatch , TRUE)
+
+/obj/item/detective_scanner/advanced
+	name = "advanced forensic scanner"
+	desc = "Processes data much quicker at the cost of not being able to scan far remotely. Gives more detailed reports. Scan from 2 tiles away to avoid leaving prints on the scene of the crime!"
```

Comment:
```suggestion
	desc = "Processes data much quicker at the cost of not being able to scan from far away. Gives more detailed reports. Scan from 2 tiles away to avoid leaving prints on the scene of the crime!"
```